code/game/machinery/requests_console.dm

```diff
@@ -329,14 +329,14 @@ var/list/obj/machinery/requests_console/allConsoles = list()
 									Console.icon_state = "req_comp2"
 								if(!Console.silent)
 									playsound(Console.loc, 'sound/machines/request.ogg', 50, 1)
-									say(text("[bicon(Console)] *The Requests Console beeps: 'Message from [department]'"))
+									say("Message from [department]")
 								Console.messages += "<B>Message from <A href='?src=\ref[Console];write=[ckey(department)]'>[department]</A></FONT></B><BR>[message]"
 
 						screen = 6
 						Console.set_light(2)
 				messages += "<B>Message sent to [dpt]</B><BR>[message]"
 			else
-				say(text("[bicon(src)] *The Requests Console beeps: 'NOTICE: No server detected!'"))
+				say("NOTICE: No server detected!")
```

Comment:
No, there is always a "thing says: " in the message, due to this shitcode it said: "the request console says: the request console beeps:"